cpp/ps0003/ps0003_soln.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+
+using namespace std;
+int main()
+{
+  // prompting user for number 2/2
+  int number;
+  cout << "please enter number \n";
+  cin >> number;
+
+  // checking if the number is odd or even 3/3
+  if (number % 2 == 0)
+    cout << "this number is even \n";
+  else
+    cout << "this number is odd\n";
```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console